src/_posts/2020-05-23-lh.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Lightning Hacks 11: Barata de salto alto não mata canhão em parede"
+layout: post
+---
+
+# Lightning Hacks 11: Barata de salto alto não mata canhão em parede
+> Dia 09/05/2020, MC: Daniel Martinez
+
+Esse dia, P2 soltou a pérola "já tentou usar uma barata pra matar um canhão?" e Caio revelou um script de filme feito por uma máquina contendo a frase
```

Comment:
Change to "[...] Caio revelou um script de filme **supostamente** feito por uma máquina contendo a frase [...]"